clang/test/Driver/sycl-nvptx-short-ptr.cpp

```diff
@@ -0,0 +1,16 @@
+// REQUIRES: nvptx-registered-target
```

Comment:
This shouldn't be needed - the test is just exercising driver behaviors, so no specific environment target should be required.

Reply:
I was wondering about that - thanks. I just copied it from another test. But as you say, all clang targets should always be available.

Anyway, fixed

Reply:
@frasercrmck, just to be clear. What Mike says is that this test doesn't use nvptx target, so it shouldn't require it. The test verifies driver's behavior only.

> all clang targets should always be available

This is not true. We build NVPTX target in this repository's CI environment, but some developers might not build this target. That's why `nvptx-registered-target` feature exists in general, but it's not relevant for this specific case.

Reply:
Right yes, sorry, that's what I meant: that no clang code is conditionally compiled depending on which LLVM targets are available. I wasn't saying that each LLVM target is always available. Unfortunately even upstream LLVM get confused about the purpose of this feature (e.g., https://github.com/llvm/llvm-project/pull/94055).